homeassistant/components/eps/strings.json

```diff
@@ -0,0 +1,21 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Connect to EPS",
+        "data": {
+          "token": "[%key:common::config_flow::data::access_token%]",
```

Comment:
From where does the user get an access token? At the very least I think a `data_description` is needed here.